geothesaurus/index.php

```diff
@@ -92,43 +95,43 @@ function toggleEditor(){
 			?>
 			<div id="updateGeoUnit-div" style="clear:both;margin-bottom:10px;">
 				<fieldset id="edit-fieldset">
-					<legend>Edit Geographic<span id="edit-legend"> Unit</span></legend>
+					<legend> <?php echo (isset($LANG['EDIT_GEO']) ? $LANG['EDIT_GEO'] : 'Edit Geographic') ?> <span id="edit-legend"> <?php echo (isset($LANG['UNIT']) ? $LANG['UNIT'] : 'Unit') ?> </span></legend>
 					<div style="float:right">
-						<span class="editIcon"><a href="#" onclick="toggleEditor()"><img class="editimg" src="../images/edit.png" /></a></span>
+						<span class="editIcon"><a href="#" onclick="toggleEditor()"><img class="editimg" src="../images/edit.png" alt="Edit"/></a></span>
```

Comment:
I've started internationalizing my `alt` text as well... when I remember to.
I think we should both just start doing this.